_posts/en/newsletters/2019-07-10-newsletter.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Bitcoin Optech Newsletter #54'
+permalink: /en/newsletters/2019/07/10/
+name: 2019-07-10-newsletter
+slug: 2019-07-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of the newest version of
+Eclair and describes a potential routing improvement for LN.  Also
+included are our regular sections about bech32 sending support and
+notable code changes in popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Eclair 0.3.1 released:** the newest version of this LN software
+  includes new and improved API calls as well other changes that allow
+  it to perform faster and use less memory.  [Upgrading][Eclair 0.3.1]
+  is recommended.
+
+- **Brainstorming just-in-time routing and free channel rebalancing:**
+  sometimes LN nodes receive a routed payment that they reject because
+  their outbound channel for that payment doesn't currently have a high
+  enough balance to support that payment.  Rene Pickhardt previously
+  [proposed][pickhardt jit] Just-In-Time (JIT) routing where the node
+  would attempt to move funds into that channel from one or more of its
+  other channel balances.  If successful, the payment could then be
+  routed; otherwise, it would be rejected like normal.  Because the
+  routed payment might fail for other reasons and prevent the routing
+  node from earning any fees, any JIT rebalance operations need to be
+  free or they could end up costing the node money in a way that
+  attackers could exploit.
+
+    In a new [post][zmn jit] to the Lightning-Dev mailing list,
+    pseudonymous LN developer ZmnSCPxj described two situations in which
+    other profit-maximizing nodes might allow free rebalances.  The
+    first case is the observation that the person at the other end of
```

Comment:
Is "node being routed through" or "downstream routing node" clearer than "the person at the other end of the outbound routing channel"?